.azure-pipelines/integration-tests.yml

```diff
@@ -275,6 +279,41 @@ jobs:
         !reproductions/**/StackExchange.Redis.AssemblyConflict.LegacyProject.csproj
       arguments: --configuration $(buildConfiguration) -p:Platform=$(buildPlatform) -p:ManagedProfilerOutputDirectory=$(publishOutput)
 
+  - task: NuGetCommand@2
+    displayName: nuget restore IIS samples
+    inputs:
+      restoreSolution: samples-iis/samples-iis.sln
+      verbosityRestore: Normal
+
+  - task: MSBuild@1
+    displayName: Publish IIS samples
+    inputs:
+      solution: samples-iis/samples-iis.sln
+      configuration: '$(buildConfiguration)'
+      msbuildArguments: '/p:DeployOnBuild=true /p:PublishProfile=FolderProfile.pubxml'
+      maximumCpuCount: true
+
+  - task: MSBuild@1
+    displayName: msbuild msi
```

Comment:
Consider changing the target in the `msbuild tracer-home` task above from `/t:CreateHomeDirectory` to `/t:msi` to do both at the same time in a single task (`msi` depends on `CreateHomeDirectory`) and removint this one.

Reply:
Good idea, attempting to do that in 22decae921078356b622db832be97a35b2cdc802. Let's see if it works...

Reply:
Yep that worked, resolving